app/controllers/prepay_balance_controller.js

```diff
@@ -0,0 +1,1557 @@
+                }
+        
+            ],
+            status: "active",
+            product: [
+                {
+                    id: faker.random.uuid(),
+                    href: faker.internet.url(),
+                    name: faker.name.findName()
+                }
+            ],
+            partyAccount: [
+                {
+                    id: faker.random.uuid(),
+                    href: faker.internet.url(),
+                    name: faker.name.findName()
+                }
+            ],
+            realizingResource: [
+                {
+                    id: faker.random.uuid(),
+                    href: faker.internet.url(),
+                    name: faker.name.findName(),
+                    value: "Resourse value"
+                }
+            ],
+            relatedParty: [
+                {
+                    id: faker.random.uuid(),
+                    href: faker.internet.url(),
+                    name: faker.name.findName(),
+                    role: "Customer"
+                }
+            ]
+        }
+    },
+    index_accumulated_balance: function(request)
+    {
+        console.log("prepay_balance_controller.index_accumulated_balance")
+
+        return [
+            {   
+                routeDetails: {
+                    controller: 'Prepay Balance',
+                    action: 'index_accumulated_balance'
+                },
+                name: faker.name.findName(),
+                description: faker.lorem.sentence(),
+                totalBalance: 
+                {
+                    amount: faker.random.number(),
+                    units: "EUR" 
+                },
+                bucket:[
+                    {
+                        id: faker.random.uuid(),
+                        href: faker.internet.url()
+                    },
+                    {
+                        id: faker.random.uuid(),
+                        href: faker.internet.url()
+                    }
+                ],
+                product: [
+                    {
+                        id: faker.random.uuid(),
+                        href: faker.internet.url(),
+                        name: faker.name.findName()
+                    }
+                ],
+                partyAccount: {
+                    id: faker.random.uuid(),
+                    href: faker.internet.url(),
+                    name: faker.name.findName()
+                },
+                relatedParty: [
+                    {
+                        id: faker.random.uuid(),
+                        href: faker.internet.url(),
+                        name: faker.name.findName(),
+                        role: "customer"
+                    }
+                ]
+            }
+        ]
+    },
+    show_accumulated_balance_products: function(request)
+    {
+        console.log("prepay_balance_controller.show_accumulated_balance_products")
+
+        return {
+                routeDetails: {
```

Comment:
missing id

Reply:
It is lower down under product id, I did fix the information though